zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -26,11 +26,7 @@
 import org.apache.zeppelin.display.AngularObject;
 import org.apache.zeppelin.display.AngularObjectRegistry;
 import org.apache.zeppelin.display.Input;
-import org.apache.zeppelin.interpreter.InterpreterFactory;
-import org.apache.zeppelin.interpreter.InterpreterGroup;
-import org.apache.zeppelin.interpreter.InterpreterResult;
-import org.apache.zeppelin.interpreter.InterpreterSetting;
-import org.apache.zeppelin.interpreter.InterpreterSettingManager;
+import org.apache.zeppelin.interpreter.*;
```

Comment:
Need to explicitly specify what class name was imported, do not use `org.apache.zeppelin.interpreter.*`

Reply:
IntelliJ IDEA automatic import package removes the asterisk (import xxx.*)

![image](https://user-images.githubusercontent.com/3677382/61890451-0cd8a780-af3a-11e9-9b25-466b7ce0ea4c.png)